setup.py

```diff
@@ -199,7 +199,7 @@ def run(self):
             *DEVELOPMENT_REQUIREMENTS,
             "scipy",  # Snowpark pandas 3rd party library testing
             "statsmodels",  # Snowpark pandas 3rd party library testing
-            "scikit-learn==1.5.2",  # Snowpark pandas scikit-learn tests
+            "scikit-learn",
```

Comment:
```suggestion
            "scikit-learn", # Snowpark pandas 3rd party library testing
```

Reply:
Accepted. GitHub is being weird about accepting suggestions.